src/window/gui/Gui.cpp

```diff
@@ -473,9 +473,15 @@ void Gui::DrawMenu() {
     if (!ImGui::DockBuilderGetNode(dockId)) {
         ImGui::DockBuilderRemoveNode(dockId);
         ImGui::DockBuilderAddNode(dockId, ImGuiDockNodeFlags_NoTabBar);
+        ImGui::DockBuilderSetNodeSize(dockId, ImVec2(viewport->Size.x, viewport->Size.y));
```

Comment:
Note that this is required so that future docks can be sized correctly. Otherwise the sizing can be inconsistent or something like that.